tensorflow_probability/examples/jupyter_notebooks/Probabilistic_Matrix_Factorization_TFP.ipynb

```diff
@@ -0,0 +1,926 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Probabilistic_Matrix_Factorization_TFP.ipynb",
+      "version": "0.3.2",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
+      "language": "python",
+      "name": "python3"
+    }
+  },
+  "cells": [
+    {
+      "metadata": {
+        "id": "fRpYIb0QraKG",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "##### Copyright 2018 The TensorFlow Authors.\n",
+        "\n",
+        "Licensed under the Apache License, Version 2.0 (the \"License\");"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "u4Hl8Q-HrZ_h",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
```

Comment:
I'm wondering if we should create a template for this (do we always need the licenses for the colab notebooks)? Also should we have authorship comments at the bottom.

@jvdillon @brianwa84 any thoughts?